tests/unit/algos/list/delimiterMaster.js

```diff
@@ -0,0 +1,179 @@
+'use strict'; // eslint-disable-line strict
+
+const assert = require('assert');
+const DelimiterMaster =
+    require('../../../../lib/algos/list/delimiterMaster').DelimiterMaster;
```

Comment:
The unit tests failed as I changed it to `require('../../../../index').algorithms.list.DelimiterMaster`. Can you shed some light here?